saleor/graphql/checkout/mutations/checkout_complete.py

```diff
@@ -87,6 +87,11 @@ class Arguments:
                 "Client-side generated data required to finalize the payment."
             ),
         )
+        metadata = NonNullList(
+            MetadataInput,
+            description="Fields required to update the object's metadata.",
```

Comment:
Missing `ADDED_IN_X`. What about `PREVIEW_FEATURE`? Do we want to mark this field with `PREVIEW_FEATURE` also?

Reply:
I wouldn't say so, it's not entirely a new feature, but rather an improvement to the existing one, what do you think @timuric ?

Reply:
@maarcingebala @korycins _preview_  is used when we expect API to change in minor versions, unless you anticipate this to happen it is not necessary

Reply:
Okay, so only ADDED_IN is missing here. cc @SzymJ Let's add the missing labels and move forward with this PR.